packages/integrations/image/src/endpoint.ts

```diff
@@ -14,7 +14,8 @@ async function loadRemoteImage(src: URL) {
 		}
 
 		return Buffer.from(await res.arrayBuffer());
-	} catch {
+	} catch (err: unknown) {
+		console.error(err)
```

Comment:
We don't report the full error, but it's currently being handled at 

https://github.com/withastro/astro/blob/eab4a0279fd28163cea9da2b010c5ccde5645e6f/packages/integrations/image/src/endpoint.ts#L35-L37

too. Are you seeing any errors that need this specific log?

Reply:
https://github.com/withastro/astro/issues/5392#issuecomment-1313425804